WcaOnRails/app/models/user.rb

```diff
@@ -155,6 +155,8 @@ def dummy_account?
     wca_id.present? && encrypted_password.blank? && email.casecmp("#{wca_id}@worldcubeassociation.org") == 0
   end
 
+  scope :delegate, -> { where.not(delegate_status: nil) }
```

Comment:
To me the plural is obviously better, but I suppose it's a personal preference rather than a universal truth.